### ISSUE TITLE
`nullref` might be changed unexpectedly while require rapidjson multi times in different lua states.

### ISSUE THREAD

**Issue by Jayatubi** [xpol/lua-rapidjson#52]
The `values::nullref ` is a static variable and would be assigned to a new value when the user requires to open rapidjson lib. However, if the user opens this lib multi times in different lua states this static vairable would be invalid to the previous lua states and cause unexpected exception.

**Comment by xpol** [comment]
Should be fixed on the master with 860c728.
Will fix GitHub actions later and make a new release.

**Comment by gottfriedleibniz** [comment]
Here's an example of `rapidjson.null` failing with the new commit:
```lua
json = require'rapidjson'
example = {
    1, 2, json.null(), 3
}

print("Main", json.encode(example))

-- false   test.lua:8: value type : function
print(coroutine.resume(coroutine.create(function()
    print("Coroutine", json.encode(example))
end)))
```

Also, if you analyze the above script with Valgrind you will notice that there's a memory leak (see http://lua-users.org/wiki/ErrorHandlingBetweenLuaAndCplusplus).

**Comment by xpol** [comment]
Should be fixed in **v0.7.0** also published to luarocks.

Note, now the type of `rapidjson.null` is not a function any. If you have ever called, remove the `()` from now on.
